### DISCUSSION TITLE
delay ON at power up

### DISCUSSION THREAD

**Discussion by johnnypeste** [arendst/Tasmota#14607] [Q&A] [answered]
Hi,

I'm a complete newbie to tasmota and I want to delay my sonoff rfr3 relay from going ON for 10 minutes after being power up.

Somewhere in this forum I saw this:

Type in console:

PowerOnState 0
rule 1
rule on system#boot do ruletimer1 600 endon on Rules#Timer=1 do power on endon

My question is so simple that I'm afraid it will be ridiculous...

How do I open the forementioned console?!

I see I can issue commands via http, for example with:

http://<ip>/cm?cmnd=Power%20On

where I presume %20 is the "space", so I should do something like:

http://<ip>/cm?cmnd=PowerOnState%200
http://<ip>/cm?cmnd=rule%201
http://<ip>/cmcmnd=rule%20on%20system#boot%20do%20ruletimer1%20600%20endon%20on%20Rules#Timer=1%20do%20power%20on&20endon

Is this correct?

And is it permanent?

**Comment by sfromis** [comment, marked as answer]
Opening the console is by clicking on the link on the Tasmota front page, open it in a browser. Better than trying to do it via such commands. (Tip: it gets much more readable by encoding space as `+` instead of `%20`, anyway).

And yes, rules are persistent across power cycling. Of course, *nothing* in this world is really *permanent*.....

**Comment by johnnypeste** [comment]
Hi
Thanks for the answer.
I think i just bricked my device
I was using a diy tool to load the firmware and it was complaining that tasmota.bin was too big.
So i loaded tasmota-minimal.bin and now i have a wifi network that i can connect but when i try to access 192.168.4.1 i can ping it but it does not have http access

Sent from Yahoo Mail on Android 
 
  On Wed, 26 Jan 2022 at 17:46, ***@***.***> wrote:   


Opening the console is by clicking on the link on the Tasmota front page, open it in a browser. Better than trying to do it via such commands. (Tip: it gets much more readable by encoding space as + instead of %20, anyway).

And yes, rules are persistent across power cycling. Of course, nothing in this world is really permanent.....

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by barbudor** [reply]
You shouldn't have flashed Tasmota-minimal. This is  a very specific version intended as an intermediate step during Tasmota upgrade process.
For Sonoff DIY you should use Tasmota-lite.bin

Not you have to open the device to do serial flashing, directly with Tasmota.bin
Should be straightforward

**Comment by johnnypeste** [comment]
Tasmota-lite.bin was also too large...
Thats why i tried minimal.
I dont have a serial converter nor soldering skills so i guess im stucked
But even if i get another device i dont know how to load a tasmota version with diy tool since the only one which is not too large is the minimal which is not good...
Dead end?

Sent from Yahoo Mail on Android 
 
  On Wed, 26 Jan 2022 at 20:10, ***@***.***> wrote:   


You shouldn't have flashed Tasmota-minimal. This is a very specific version intended as an intermediate step during Tasmota upgrade process.
For Sonoff DIY you should use Tasmota-lite.bin

Not you have to open the device to do serial flashing, directly with Tasmota.bin
Should be straightforward

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by barbudor** [reply]
I've heard that people have been successfull with [tasmota-lite v9.3.1](http://sidweb.nl/tasmota/release-9.1.0/) which is 30k less

I don't know the Sonoff RF R3 but it is often very simple to serial flash. May need a 3rd hand to help. 
There are instructions here https://vivio.wordpress.com/2020/04/23/sonoff-rfr3-433-mqtt-gateway-mod/
With a USB/Serial adapter like the recommended [CH340G Golden](https://www.aliexpress.com/item/1005001867665663.html) and some "Dumont" cables its should be easy

**Comment by johnnypeste** [comment]
You mean 9.1.0 ...I will try. Thanks

Sent from Yahoo Mail on Android 
 
  On Wed, 26 Jan 2022 at 20:51, ***@***.***> wrote:   


I've heard that people have been successfull with tasmota-lite v9.3.1 which is 30k less

I don't know the Sonoff RF R3 but it is often very simple to serial flash. May need a 3rd hand to help.
There are instructions here https://vivio.wordpress.com/2020/04/23/sonoff-rfr3-433-mqtt-gateway-mod/
With a USB/Serial adapter like the recommended CH340G Golden and some "Dumont" cables its should be easy

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by johnnypeste** [comment]
Success.
I offered the sonoff device to someone with soldering skills and loaded the older tasmota-lite you said on the other device...
It works fine but the 10 minutes is after the connection with the router and not at power on. If the router is disconnected when i turn the relay on nothing happens.,.
Thanks...


Sent from Yahoo Mail on Android 
 
  On Wed, 26 Jan 2022 at 21:05, Joao ***@***.***> wrote:   You mean 9.1.0 ...I will try. Thanks

Sent from Yahoo Mail on Android 
 
  On Wed, 26 Jan 2022 at 20:51, ***@***.***> wrote:   


I've heard that people have been successfull with tasmota-lite v9.3.1 which is 30k less

I don't know the Sonoff RF R3 but it is often very simple to serial flash. May need a 3rd hand to help.
There are instructions here https://vivio.wordpress.com/2020/04/23/sonoff-rfr3-433-mqtt-gateway-mod/
With a USB/Serial adapter like the recommended CH340G Golden and some "Dumont" cables its should be easy

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by barbudor** [reply]
When serial flashing, you can use directly the lastest Tasmota.bin. My suggestion of tasmota-lite v9.1.0 was only for Sonoff DIY
Nevertheless, you can now upgrade to latest tamota.bin.gz
The easiest and simplest way is to enter the URL http://ota.tasmota.com/tasmota/release/tasmota.bin.gz in the the 1 part of the Upgrade menu and let Tasmota download the file.
Remeber that for serial flashing you should use the files ending in `.bin` but for anything like OTA or File Upload you should use the `.bin.gz` without trying to decompress in advance. Tasmota will decompress on chip.

Look at the available triggers there : https://tasmota.github.io/docs/Rules/#rule-trigger
There are different Xxxx#Boot available which triggers at different point in time. Definitively System#Boot is the latest that occurs when Tasmota is fully initialized and connected.
The earliest trigger is Power1#Boot and will occur even without Wifi connection
